fhttp/httprunner_test.go

```diff
@@ -119,10 +119,10 @@ func TestServe(t *testing.T) {
 	if err != nil {
 		log.Fatalf("Unable to listen to dynamic port: %v", err)
 	}
-	port := listener.Addr().(*net.TCPAddr).Port
-	log.Infof("Using port: %d", port)
+	port := fmt.Sprintf("%d", (listener.Addr().(*net.TCPAddr).Port))
```

Comment:
apologies for all the back and forth but
there is no value in changing this part / the test from int to string, it's doing the right thing which is extracting the numerical port (which we should do elsewhere in non test code) can you revert that change/file ?

Reply:
`port` get passed to the `Serve` function in line 127:
```
go func() {
	Serve(port, "/debugx1")
}()
```

The `Serve` func now takes a string instead of an int, so I need to change the type.